### ISSUE TITLE
Oficinas de tecnologia para o NETI

### ISSUE THREAD

**Issue by monfardineL** [casinufsc/ideias#4]
[Essa ideia veio de uma apresentação sobre Smart Cities para a matéria de Habitats de Inovação]

O [NETI](http://neti.ufsc.br/) serve de referência para estudos de graduação e pós-graduação. Com enfoque na educação permanente, oferece cursos, grupos, oficinas e projetos voltados para alunos idosos, objetivando a sua atualização e inserção social. O Núcleo também presta assessoria e consultoria à comunidade, através de parcerias com entidades governamentais e não governamentais.

Durante a disciplina foram levantados pontos como a dificuldade do público idoso na interação com a tecnologia. Parece já ter havido uma turma para ensinar como utilizar _smartphone_ por exemplo, porém essa atividade não aparece mais listada no site deles.  E surgiu a ideia do desenvolvimento de oficinas para instrução e auxílio desse público, garantindo melhor interação com os conceitos de _Smart Cities._
A proposta seria oferecer cursos de interação com tecnologia, de acordo com necessidades levantadas pelo próprio núcleo, que seriam ministrados pelos alunos do curso. Em contrapartida, a universidade oferece a validação de horas de atividades complementares.

**Comment by matheuslc** [comment]
@monfardineL cara, achei muito irado essa ideia! A gente poderia mapear quais aplicativos/serviços esses grupos mais utilizam e ensinar eles a usar. Isso tem potencial de ser muito positivo para esse público, principalmene se que com isso eles consigam se comunicar melhor.

**Comment by monfardineL** [comment]
@matheuslc sim, o passo inicial seria juntar um grupo que esteja disposto, formalizar a ideia e levar até o NETI, pedindo o auxilio deles para identificar o que é mais necessário.
(Ps: não consigo adicionar o label de ideia, não sei se é pq sou noob de github).

**Comment by sabrinaschutz** [comment]
Gostei bastante, já digo que estou disposta a ajudar haha

**Comment by matheuslc** [comment]
Massa! Estou disposto também. Esse inclusive pode ser um projeto recorrente e que pode acontecer em paralelos a outros projetos, né?
